darwin/types.py

```diff
@@ -0,0 +1,6 @@
+from typing import Dict, List
+
+Point = Dict[str, float]
+BoundingBox = Dict[str, float]
```

Comment:
`TypedDict` would have been better, but it's only available in typing since Python 3.8